cosmos/loadtest/scripts/loadtest.sh

```
@@ -0,0 +1,130 @@
+  update_config
+  json_processor "$APP_HOME/config/genesis.json" '.genesis_time = "'"$(current_time)"'"'
+  start_node
+  node_catching_up "" "" || exit 1
+  ./loadtest "http://127.0.0.1:26657" "./config.json" >"$APP_HOME/loadtest.log" 2>&1
+  avg_block_time_interval
+  kill -9 "$(cat "$APP_HOME/app.pid")"
+  sleep 10
+}
+
+function run_fxcore() {
+  export APP_BIN=$(which fxcored 2>/dev/null)
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)